Intersect (Core)/Attributes/Attributes.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Enums;
+
+namespace Intersect.Attributes;
+internal class Attributes
```

Comment:
Uh, why are these "public" classes wrapped in another class at all, but especially one that is "internal"?